src/main/resources/templates/EditQuestion.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <title>Based Survey</title>
+
+  <style>
+    table, th, td {
+      border: 1px solid;
+    }
+  </style>
+</head>
+<body>
+  <h1 th:text="'Question for Survey with ID:'+${survey.getId()}">Survey Info</h1>
```

Comment:
optional, but it would be nice if this was a link to go back to my survey edit part

Reply:
Good idea for the alpha release